kyo-prelude/shared/src/main/scala/kyo2/Env.scala

```diff
@@ -33,7 +33,7 @@ object Env:
         inline def apply[A, S](inline f: R => A < S)(
             using tag: Tag[R]
         ): A < (Env[R] & S) =
-            ContextEffect.suspend(erasedTag[R]) { map =>
+            ContextEffect.suspendMap(erasedTag[R]) { map =>
```

Comment:
A minor unrelated refactoring: I've changed `ContextEffect` to use `suspendMap` as the name for the methods that suspend with a continuation. It follows the method naming in `Effect`.